conf/test.config

```diff
@@ -97,18 +107,12 @@ profiles {
     }
 }
 
+//This is apparently useless as it won't overwrite things in the modules.config
```

Comment:
yes, this is because `test.config` is loaded before `modules.config`. you want it to overwrite the `modules.config` it needs to be passed as a parameter there, e.g. a new param called `--single_fork` that is set on the test profile, and if this one is set then the `modules.config` sets the maxForks to 1 of SNPEFF and ENSEMBLVEP.

Reply:
Yes figured it out with Mahesh's help :D I was pretty confused for about an hour there 😆